src/Rotations.jl

```diff
@@ -48,6 +53,8 @@ export
         valid_rotation,
 
         # extra quaternion operations
-        rot_angle  # N.B. angle(Quaternions) is different and defined in Quaternions
+        rot_angle,  # N.B. now depricated in favour of angle
```

Comment:
Did we ever figure out why Quaternions defines this oddly?  Or is Quaternions fixed (in which case a minimum version should go in REQUIRE)?